### DISCUSSION TITLE
what dose Time < T_lag ? 0 : k_a*A_ET mean (Expression in Mobi or PKSIM)

### DISCUSSION THREAD

**Discussion by hainiwen** [Open-Systems-Pharmacology/Forum#1532] [Questions & Problems] [answered]
Hi there!
What dose this expression ? 0 : means in Mobi?
I think it's something related to conditions, but I don't know what language is that
Can anybody explain a bit?

**Comment by Yuri05** [comment, marked as answer]
https://docs.open-systems-pharmacology.org/working-with-mobi/mobi-documentation/model-building-components#working-with-formulas
It's a conditional expression, using the notation `<condition> ? <formula string for true> : <formula string for false>`

So `Time < T_lag ? 0 : k_a*A_ET` means:
```
IF Time < T_lag
  0
ELSE
  k_a*A_ET
```
